<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DS1000Z Simple Power Analisis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PlotWidget" name="graph"/>
        </item>
        <item>
         <widget class="QTableWidget" name="table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Width</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Notes</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonChan1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string> CHAN1</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/chan1.png</normaloff>icons/chan1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonChan2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string> CHAN2</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/chan2.png</normaloff>icons/chan2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonChan3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string> CHAN3</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/chan3.png</normaloff>icons/chan3.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonChan4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string> CHAN4</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/chan4.png</normaloff>icons/chan4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Capture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="captureList">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionDownload"/>
   <addaction name="separator"/>
   <addaction name="actionSingle"/>
   <addaction name="actionTriggerLoop"/>
   <addaction name="separator"/>
   <addaction name="actionRange"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionExportMarkers"/>
   <addaction name="actionClearMarkers"/>
  </widget>
  <action name="actionDownload">
   <property name="icon">
    <iconset>
     <normaloff>icons/cloud-arrow-down.png</normaloff>icons/cloud-arrow-down.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Get data from scope</string>
   </property>
  </action>
  <action name="actionMarker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/marker.png</normaloff>icons/marker.png</iconset>
   </property>
   <property name="text">
    <string>Marker</string>
   </property>
   <property name="toolTip">
    <string>Create a marking</string>
   </property>
  </action>
  <action name="actionRange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/arrows-left-right-to-line.png</normaloff>icons/arrows-left-right-to-line.png</iconset>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
   <property name="toolTip">
    <string>Select a range</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>icons/rotate-left.png</normaloff>icons/rotate-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo marking</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>icons/scissors.png</normaloff>icons/scissors.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut selected range</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icons/folder-open.png</normaloff>icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClearMarkers">
   <property name="icon">
    <iconset>
     <normaloff>icons/broom.png</normaloff>icons/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear Markers</string>
   </property>
  </action>
  <action name="actionExportMarkers">
   <property name="icon">
    <iconset>
     <normaloff>icons/file-export.png</normaloff>icons/file-export.png</iconset>
   </property>
   <property name="text">
    <string>Export Markers</string>
   </property>
  </action>
  <action name="actionSingle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Trigger Single</string>
   </property>
  </action>
  <action name="actionTriggerLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/repeat.png</normaloff>icons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>Trigger Loop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
